### PR TITLE
mark processor down for some time not updating resolve ts

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
close the old pr https://github.com/pingcap/ticdc/pull/159 since I can't update it after make repo public.

mark processor down for some time not updating resolve ts

### What is changed and how it works?
mark processor down for some time not updating resolve ts.
and clear up data dispatch tables to other captures laster.

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### CODE REVIEW

File: cdc/owner.go [hunk 1]
```diff
@@ -384,7 +431,26 @@ func (o *ownerImpl) loadChangeFeedInfos(ctx context.Context) error {
...
 
 		if cfInfo, exist = o.changeFeedInfos[changeFeedID]; exist {
+			for cid, pinfo := range etcdChangeFeedInfo {
+				if _, ok := cfInfo.processorLastUpdateTime[cid]; !ok {
+					cfInfo.processorLastUpdateTime[cid] = time.Now()
+					continue
+				}
+
+				oldPinfo, ok := cfInfo.ProcessorInfos[cid]
+				if !ok || oldPinfo.ResolvedTs != pinfo.ResolvedTs || oldPinfo.CheckPointTs != pinfo.CheckPointTs {
```
Comment (july2993) [inline cdc/owner.go]: update to check both @amyangfei

File: cdc/owner.go [hunk 2]
```diff
@@ -34,6 +35,8 @@ import (
 	"go.uber.org/zap"
 )
 
+var markProcessorDownTime = 5 * time.Minute
```
Comment (july2993) [inline cdc/owner.go]: @amyangfei just increase to be 5 minute now.
I think the better implementation should check the aliveness of capture, like maintain a session in etcd.

Comment (amyangfei) [reply cdc/owner.go]: Oh I mistake it at first, it is one minute, not one second. Five minutes is too long for CDC owner to detect processor down 🙁️
maybe we can recover it to 1 minute for the test, and have a better detect way in other PR.

Comment (july2993) [reply cdc/owner.go]: reverted

**Pull request merged**